src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const listOfEmployees = [...document.querySelectorAll(`li`)];
```

Comment:
Ensure that the `listOfEmployees` is correctly populated with the intended list items. If the `li` elements are not direct children of the document, this query might not work as expected.